naurffxiv/src/app/uwu.js

```diff
@@ -1,5 +1,69 @@
 export default function DisplayUWU() {
 	return (
-		<p>Placeholder: Put uwu resources here</p>
+		<>
+		<h1>UWU Strats and Resources</h1>
+			<h2>Table of Contents</h2>
+				<ol>
+					<li>UWU Strats by Phases</li>
+					<li>General Fight Resources</li>
+					<li>UWU Visual Guides and POVs</li>
+				</ol>
+					<h3>UWU Strats By Phase</h3>
+						<h4>Phase 1: Garuda</h4>
+							<p><a href="https://ff14.toolboxgaming.space/?id=359535529826261&preview" target="_blank">Garuda Toolbox</a></p>
+							<p><em>Please note that the caster goes east and the off-tank goes west to take Mesohigh tethers.</em></p>
+						<h4>Phase 2: Ifrit</h4>
+							<p><a href="https://ff14.toolboxgaming.space/?id=562530446784261&preview" target="_blank">Ifrit Toolbox</a></p>
+							<p><em>Ifrit Dashes can be skipped with sufficiently high DPS.</em></p>	
+						<h4>Phase 3: Titan</h4>
+							<p><a href="https://ff14.toolboxgaming.space/?id=463534803605261&preview" target="_blank">Titan Toolbox</a></p>
+							<p><em>Party Finder groups in North America and Europe expect the use of an "automarker" (AM, "Allagan Melon") third party tool to simplify the Titan Gaols mechanic. There are Terms of Service-friendly ways of resolving this mechanic (e.g., self-marking macros), although such use is extremely rare in Party Finder and more suitable in a static environment.</em></p>	
+						<h4>Intermission</h4>
+							<p><a href="https://ff14.toolboxgaming.space/?id=430631425646261&preview" target="_blank">Intermission Toolbox</a></p>
+						<h4>Phase 4: Ultima</h4>
+							<p><a href="https://ff14.toolboxgaming.space/?id=530635345646261&preview" target="_blank">Predation Toolbox</a></p>
+							<p><a href="https://ff14.toolboxgaming.space/?id=930637786646261&preview" target="_blank">Annihilation Toolbox </a></p>
+							<p><a href="https://ff14.toolboxgaming.space/?id=830635036646261&preview" target="_blank">Suppression Toolbox</a></p>
+							<p><em>Tanks left, party right on aetheric booms.</em></p>	
+					<h3>General fight resources</h3>
+						<p><a href="https://docs.google.com/spreadsheets/d/1V-FXFOW8pj87DtqT4KpKSbu3AZYI4diSjH1F-FyNt_o/edit?usp=sharing" target="_blank">Sample Mitigation Sheet</a></p>
+						<p><a href="https://docs.google.com/spreadsheets/d/13dpeUEBHFpLxX38gjC8oh1xEB4F7xBECI9BvycJEsfk/edit#gid=831820397" target="_blank">Sample Tank Mitigation Sheet</a></p>
+						<p><a href="https://github.com/naurffxiv/assets/blob/main/Toolbox%20Resources/UWU/TitanGaolsCheatSheet.png?raw=true" target="_blank">Titan Gaols Cheat Sheet</a></p>
```

Comment:
you don't have to wrap all of these in `<p>` tags, they should be fine as left one `<a href...>` or `<em>` tags

Reply:
Addressed in #8